modules/post/multi/manage/screenshare.rb

```diff
@@ -0,0 +1,174 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Multi Manage the screen of the target meterpreter session',
+        'Description' => %q{
+          This module allows you to view and control the screen of the target computer via
+          a local browser window. The module continually screenshots the target screen and
+          also relays all mouse and keyboard events to session.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [ 'timwr'],
+        'Platform' => [ 'linux', 'win' ],
```

Comment:
I've confirmed this working with OSX and Windows, but there was a warning saying that my osx session may not be compatible with this module

```
msf5 post(multi/manage/screenshare) > run

[!] SESSION may not be compatible with this module.
[*] Using URL: http://127.0.0.1:8081/c2B9eKruu8kWG5Q
[*] Server started.
[*] Sent screenshare html to 127.0.0.1
```